JS/Lesson34/goksel/recipe-app/src/App.js

```diff
@@ -1,23 +1,29 @@
 import { useState, useEffect } from 'react';
-import { Search } from './modules/recipes/SearchBar';
-import './App.css';
+
+import { AppRouter } from './AppRouter';
+import { Navbar } from './components/Navbar';
 
 export const App = () => {
 	const [recipes, setRecipes] = useState([]);
+	const [isSignedIn, setIsSignedIn] = useState(true);
 
 	useEffect(() => {
 		const DB_URL =
 			'https://www.themealdb.com/api/json/v1/1/filter.php?c=Seafood';
 		fetch(DB_URL)
 			.then((data) => data.json())
-			.then((recipeData) => 
-			setRecipes(recipeData.meals));
+			.then((recipeData) =>
+				setRecipes(recipeData.meals)
+			);
 	}, []);
 
+
 	return (
-		<div className={`app ${bgColor}`}>
-      <Search setRecipes={setRecipes}/>
-			<ul className='recipes__list'>{recipes.map((recipe => <li key={recipe.idMeal} >{recipe.strMeal}</li>))}</ul>
-		</div>
+		<>
+			<Navbar setIsSignedIn={setIsSignedIn} isSignedIn={isSignedIn} />
+			<div className="appRouter" >
```

Comment:
It's better to rename this class to something like 'app-container' since router is usually an element not focused on the visual representation but on functionality.
```suggestion
			<div className="app-container" >
```

Reply:
Renamed